utils.bal

```diff
@@ -0,0 +1,114 @@
+    string encodedParamStrValue = check url:encode(paramString, UTF_8);
+    string encodedConsumerSecretValue = check url:encode(consumerSecret, UTF_8);
+    string encodedAccessTokenSecretValue = check url:encode(accessTokenSecret, UTF_8);
+
+    //Create a signature
+    string baseString = httpMethod + "&" + encodedServiceEPValue + "&" + encodedParamStrValue;
+    byte[] baseStringByte = baseString.toBytes();
+    string keyStr = encodedConsumerSecretValue + "&" + encodedAccessTokenSecretValue;
+    byte[] keyArrByte = keyStr.toBytes();
+    string signature = (check crypto:hmacSha1(baseStringByte, keyArrByte)).toBase64();
+
+    string encodedSignatureValue = check url:encode(signature, UTF_8);
+    string encodedaccessTokenValue = check url:encode(accessToken, UTF_8);
+
+    string oauthHeaderString = "OAuth oauth_consumer_key=\"" + consumerKey +
+        "\",oauth_signature_method=\"HMAC-SHA1\",oauth_timestamp=\"" + timeStamp +
+        "\",oauth_nonce=\"" + nonce + "\",oauth_version=\"1.0\",oauth_signature=\"" +
+        encodedSignatureValue + "\",oauth_token=\"" + encodedaccessTokenValue + "\"";
+    request.setHeader("Authorization", regex:replaceAll(oauthHeaderString, "\\\\", ""));
+    return ();
+}
+
+function createRequestHeaderMap(http:Request request, string httpMethod, string resourcePath, string consumerKey,
+        string consumerSecret, string accessToken, string accessTokenSecret, string paramStr) returns map<string>|error {
+    string serviceEndpoint = "https://api.twitter.com" + resourcePath;
+    string paramString = paramStr.substring(0, paramStr.length() - 1);
+    string encodedServiceEPValue = check url:encode(serviceEndpoint, UTF_8);
+    string encodedParamStrValue = check url:encode(paramString, UTF_8);
+    string encodedConsumerSecretValue = check url:encode(consumerSecret, UTF_8);
+    string encodedAccessTokenSecretValue = check url:encode(accessTokenSecret, UTF_8);
+
+    string baseString = httpMethod + "&" + encodedServiceEPValue + "&" + encodedParamStrValue;
+    byte[] baseStringByte = baseString.toBytes();
+    string keyStr = encodedConsumerSecretValue + "&" + encodedAccessTokenSecretValue;
+    byte[] keyStringByte = keyStr.toBytes();
+    string signature = (check crypto:hmacSha1(baseStringByte, keyStringByte)).toBase64();
+
+    string encodedSignatureValue = check url:encode(signature, UTF_8);
+    string encodedaccessTokenValue = check url:encode(accessToken, UTF_8);
+
+    string oauthHeaderString = "OAuth oauth_consumer_key=\"" + consumerKey +
+        "\",oauth_signature_method=\"HMAC-SHA1\",oauth_timestamp=\"" + timeStamp +
+        "\",oauth_nonce=\"" + nonce + "\",oauth_version=\"1.0\",oauth_signature=\"" +
+        encodedSignatureValue + "\",oauth_token=\"" + encodedaccessTokenValue + "\"";
+    // request.setHeader("Authorization", regex:replaceAll(oauthHeaderString, "\\\\", ""));
+    // return map<string> headerMap;
```

Comment:
Shall we remove commented out code segment above?

Reply:
Removed